### ISSUE TITLE
SSO / Element X

### ISSUE THREAD

**Issue by aparrot89** [spantaleev/matrix-docker-ansible-deploy#3395]
Is it possible to use the sso keycloak (oidc/saml) with the Element X client ?

**Comment by ZelChief** [comment]
Yes, they [recommended](https://github.com/element-hq/element-meta/issues/1915#issuecomment-2205411288) to use [matrix-authentication-service](https://github.com/matrix-org/matrix-authentication-service).

@spantaleev, I use [matrix_synapse_oidc_providers](https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/master/docs/configuring-playbook-synapse.md#synapse--openid-connect-for-single-sign-on) for standard clients. Сan you help/add support matrix-authentication-service?

**Comment by ZelChief** [comment]
UPD: https://github.com/element-hq/element-meta/issues/1915#issuecomment-2304903978

**Comment by NickBouwhuis** [comment]
> Yes, they [recommended](https://github.com/element-hq/element-meta/issues/1915#issuecomment-2205411288) to use [matrix-authentication-service](https://github.com/matrix-org/matrix-authentication-service).

Since [matrix-authentication-service](https://github.com/matrix-org/matrix-authentication-service) has been deprecated. Can you recommend anything else?

**Comment by NickBouwhuis** [comment]
Are there any plans to include [matrix-authentication-service](https://github.com/matrix-org/matrix-authentication-service)? I'd contribute myself if I had more time and/or experience with Matrix. Or anyone that has already deployed [matrix-authentication-service](https://github.com/matrix-org/matrix-authentication-service) separately and give us some pointers?

**Comment by spantaleev** [comment]
matrix-authentication service has been supported since 2024-10-19.

See the [Support for Matrix Authentication Service](https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/75337384c00c664ad8a6d3982d9fb2f328b9e575/CHANGELOG.md#support-for-matrix-authentication-service) changelog entry.

**Comment by NickBouwhuis** [comment]
Awesome. Totally missed that. Thanks!
